src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -223,6 +231,15 @@ private void resendStaffNotificationsOnClaimIssued(Claim claim, String authorisa
 
     }
 
+    private void resendStaffNotificationForIntentToProceed(Claim claim) {
+        ClaimantResponse claimantResponse = claim.getClaimantResponse().orElseThrow(IllegalArgumentException::new);
+
+        if (directionsQuestionnaireEnabled && claimantResponse.getType() == REJECTION) {
```

Comment:
Would it be nice to send some useful information back in response if this condition is false?

Reply:
Also should we use `uk.gov.hmcts.cmc.claimstore.utils.ClaimantResponseHelper#isIntentToProceed` instead of `claimantResponse.getType() == REJECTION`?